js/webpack.config.js

```diff
@@ -131,6 +132,16 @@ module.exports = {
         ]
       }),
       new HappyPack({
+        id: 'less',
```

Comment:
Agreed for using PostCSS, just wanted to try out.
And LESS is automatically loaded by Webpack, from the `less` and `less-loader` packages.